app/controllers/bulk_upload_lettings_logs_controller.rb

```diff
@@ -3,7 +3,9 @@ class BulkUploadLettingsLogsController < ApplicationController
   before_action :validate_data_protection_agrement_signed!
 
   def start
-    if have_choice_of_year?
+    if current_user.support? && params[:organisation_id].present?
+      redirect_to bulk_upload_lettings_log_path(id: "upload-your-file", form: { year: current_year }, organisation_id: params[:organisation_id])
```

Comment:
I've reworked this to also go back to the year page should it be needed, the support users will not skip to the upload your file page, and will begin on /start. If a year needs picking then they'll go to the year page otherwise the prepare your file page.